README.md

```diff
@@ -299,6 +300,28 @@ In order to rotate your log files, add in `/etc/logrotate.d/myapp`:
 }
 ```
 
+<a name="transports"></a>
+## What happened to my transports?
+
+There will never be an API for transports, or support for ObjectMode Writable streams.
+This library is fast because it does way less than the others. We went a
```

Comment:
`We went a great length` doesn't read right, I think it should be `We went to great lengths`